main.go

```diff
@@ -75,23 +103,23 @@ func (cmd *updateConfigCommand) run() error {
 		opt = helm.ReuseValues(true)
 	}
 
-	newValues, _ := yaml.Marshal(dest)
 	_, err = cmd.client.UpdateReleaseFromChart(
 		ls.Releases[0].Name,
 		ls.Releases[0].Chart,
-		helm.UpdateValueOverrides(newValues),
+		helm.UpdateValueOverrides(rawVals),
```

Comment:
And would you please help confirm, after the update, you can still get the full set of the values. It seems that the input of helm.UpdateValueOverrides is the values you need to update, the final result will only have these values, and lose the original ones.

Reply:
I just fetched values thou `helm get values cf.20190131.134059  --all`  and got all values including what I didn't update.